### PR TITLE
Filter list of serial ports based on USB PID and VID

### PR DESCRIPTION
**Pull request by hubertmis**

### CODE REVIEW

**Comment by hubertmis** [comment]
@LuDuda No, I don't have Windows or Mac at home. Tried only with Linux.

**Comment by stig-bjorlykke** [comment]
This works on macOS.

**Comment by hubertmis** [comment]
@e-rk Yep, I've used this document as guide and reference for implementation

File: nrf802154_sniffer/nrf802154_sniffer.py [hunk 1]
```diff
@@ -63,6 +63,10 @@ class Nrf802154Sniffer(object):
     DLT='802.15.4'
     DLT_NO = 147 if DLT == 'user' else 230
 
+    # USB device identification.
+    NORDICSEMI_VID = 0x1915
+    SNIFFER_802154_PID = 0x0154
```
Comment (LuDuda) [inline nrf802154_sniffer/nrf802154_sniffer.py]: Is PID finally 0x0154 or 0x1540? I think the latter?

Comment (hubertmis) [reply nrf802154_sniffer/nrf802154_sniffer.py]: It's `0x0154` in the firmware I downloaded from this repo. I reserved `0x1540` for the sniffer purpose, but someone changed my reservation to `0x0154`. Don't know who ;)

Comment (LuDuda) [reply nrf802154_sniffer/nrf802154_sniffer.py]: It has been reverted. As I mentioned in my previous comment, the hex file should be changed and this PR accepted now.

**Pull request merged**